docs/paper/dev/misc/databases.md

```diff
@@ -0,0 +1,107 @@
+---
+slug: dev/using-databases
+---
+
+# Using Databases
+
+When you are storing larger amounts of data inside a plugin, we recommend using a database. This guide will walk you through the startup process.
+
+## MySQL vs SQLite
+
+There are two main types of database software that you could use: MySQL vs SQLite. They both have many pros and cons outlined [here](https://www.hostinger.com/tutorials/sqlite-vs-mysql-whats-the-difference/)
```

Comment:
this is meh, instead of linking products, we should talk about file based dbs (like sqlite, h2, whatever) vs standalone dbs (postgres, mysql, etc).
that article in particular is worthless seo spam